pom.xml

```diff
@@ -103,11 +98,11 @@
     <distributionManagement>
         <snapshotRepository>
             <id>ion-snapshots</id>
-            <url>http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-snapshots/</url>
+            <url>http://artifacts.codice.org/content/repositories/ion-snapshots/</url>
         </snapshotRepository>
         <repository>
             <id>ion-releases</id>
-            <url>http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/</url>
+            <url>https://artifacts.codice.org/content/repositories/ion-releases/</url>
```

Comment:
✏️ Nitpick: Is there a difference in using "https" here instead of "http" as in the `ion-snapshots` section above?

Reply:
The "https" improves security. It ensures that the integrity of the artifacts.

Reply:
Security is always good. Would it then make sense to do the same for the `ion-snapshots` URL too?